src/pyfaf/actions/opsysdel.py

```diff
@@ -25,28 +25,29 @@ class OpSysDel(Action):
 
 
     def run(self, cmdline, db):
-        opsys = get_opsys_by_name(db, cmdline.NAME)
+        for opsys in cmdline.OPSYS:
+            db_opsys = get_opsys_by_name(db, opsys)
 
-        if not opsys:
-            self.log_error("Operating system '{0}' not found"
-                           .format(cmdline.NAME))
-            return 1
+            if not db_opsys:
+                self.log_error("Operating system '{0}' not found"
+                               .format(db_opsys))
+                continue
```

Comment:
Is it an error then if we proceed anyway? Would a warning above be more appropriate if the error is not fatal?

Reply:
Changed to warning.